Disconnected service tests failing

Tests failing due to changed naming conventions.



None
name:	test_nosegid_listener_timeout
description:	None
result:	fail
failure msg:	setup_l2adjacent_test = <logging.FileHandler object at 0x7f7850c3a610>
bigip = <f5.bigip.ManagementRoot object at 0x7f784fb284d0>

    def test_nosegid_listener_timeout(setup_l2adjacent_test, bigip):
        def create_mock_rpc_plugin():
            mock_rpc_plugin = mock.MagicMock(name='mock_rpc_plugin')
            mock_rpc_plugin.get_port_by_name.return_value = [
                {'fixed_ips': [{'ip_address': '10.2.2.134'}]}
            ]
            mock_rpc_plugin.get_all_loadbalancers.return_value = [
                {'lb_id': u'50c5d54a-5a9e-4a80-9e74-8400a461a077'}
            ]
            service = deepcopy(NOSEGID_CREATELISTENER)
            service['loadbalancer']['provisioning_status'] = "ACTIVE"
            mock_rpc_plugin.get_service_by_loadbalancer_id.return_value = service
            return mock_rpc_plugin
        # Configure
        icontroldriver, start_registry = handle_init_registry(
            bigip, FEATURE_ON, create_mock_rpc_plugin)
        gtimeout = icontroldriver.conf.f5_network_segment_gross_timeout
        poll_interval = icontroldriver.conf.f5_network_segment_polling_interval
        service = deepcopy(NOSEGID_CREATELISTENER)
        logcall(setup_l2adjacent_test,
                icontroldriver._common_service_handler,
                service)
        # Set timers
        start_time = time.time()
        timeout = start_time + gtimeout
        # Begin operations
        while time.time() < (timeout + (2*poll_interval)):
            time.sleep(poll_interval)
            create_registry = register_device(bigip)
            create_uris = set(create_registry.keys()) - set(start_registry.keys())
>         assert create_uris == SEG_INDEPENDENT_LB_URIS | NOSEG_LISTENER_URIS
E         assert set(['https:/...?ver=12.1.0']) == set(['https://...?ver=12.1.0'])
E             Extra items in the left set:
E             'https://localhost/mgmt/tm/ltm/virtual/~TEST_128a63ef33bc4cf891d684fad58e7f2d~TEST_105a227a-cdbf-4ce3-844c-9ebedec849e9?ver=12.1.0'
E             'https://localhost/mgmt/tm/ltm/virtual-address/~TEST_128a63ef33bc4cf891d684fad58e7f2d~TEST_50c5d54a-5a9e-4a80-9e74-8400a461a077?ver=12.1.0'
E             Extra items in the right set:
E             'https://localhost/mgmt/tm/ltm/virtual/~TEST_128a63ef33bc4cf891d684fad58e7f2d~SAMPLE_LISTENER?ver=12.1.0'
E             'https://localhost/mgmt/tm/ltm/virtual-address/~TEST_128a63ef33bc4cf891d684fad58e7f2d~10.2.2.140?ver=12.1.0'
E             Full diff:
E             - set([u'https://localhost/mgmt/tm/ltm/virtual-address/~TEST_128a63ef33bc4cf891d684fad58e7f2d~TEST_50c5d54a-5a9e-4a80-9e74-8400a461a077?ver=12.1.0',
E             ?                                                                                             ^^^^^^ ^^^^^^^^^^^^^^^^^^^^^^^ ---------
E             Detailed information truncated (10 more lines), use "-vv" to show

test_disconnected_service_creation.py:419: AssertionError

--- a/test/functional/neutronless/disconnected_service/test_disconnected_service_creation.py
+++ b/test/functional/neutronless/disconnected_service/test_disconnected_service_creation.py
@@ -109,20 +109,20 @@ SEG_DEPENDENT_LB_URIS =\
 SEG_LISTENER_URIS = \
     set([u'https://localhost/mgmt/tm/ltm/virtual-address/'
          '~TEST_128a63ef33bc4cf891d684fad58e7f2d'
-         '~10.2.2.140%251?ver='+tmos_version,
+         '~TEST_50c5d54a-5a9e-4a80-9e74-8400a461a077?ver='+tmos_version,
 
          u'https://localhost/mgmt/tm/ltm/virtual/'
          '~TEST_128a63ef33bc4cf891d684fad58e7f2d'
-         '~SAMPLE_LISTENER?ver='+tmos_version])
+         '~TEST_105a227a-cdbf-4ce3-844c-9ebedec849e9?ver='+tmos_version])
 
 NOSEG_LISTENER_URIS =\
     set([u'https://localhost/mgmt/tm/ltm/virtual-address/'
          '~TEST_128a63ef33bc4cf891d684fad58e7f2d'
-         '~10.2.2.140?ver='+tmos_version,
+         '~TEST_50c5d54a-5a9e-4a80-9e74-8400a461a077?ver='+tmos_version,
 
          u'https://localhost/mgmt/tm/ltm/virtual/'
          '~TEST_128a63ef33bc4cf891d684fad58e7f2d'
-         '~SAMPLE_LISTENER?ver='+tmos_version])
+         '~105a227a-cdbf-4ce3-844c-9ebedec849e9?ver='+tmos_version])
 
 ERROR_MSG_MISCONFIG = 'Misconfiguration: Segmentation ID is missing'
 ERROR_MSG_VXLAN_TUN = 'Failed to create vxlan tunnel:'

